packages/techdocs-addons/src/index.ts

```diff
@@ -28,3 +28,4 @@ export {
 } from './addons';
 export { TechDocsAddonLocations } from './types';
 export type { TechDocsAddonAsyncMetadata, TechDocsAddonOptions } from './types';
+export * from './test-utils';
```

Comment:
By importing and exporting `test-utils` here we drag in test only files that rely on `jest` and some mocks that automatically get activated on import, hence I think we need to separate import entry points.

One option would be a different package (might not be preferred) or find a way to export it as e.g. `@backstage/techdocs-addons/test-utils`

Reply:
I'm not sure I understand, we just export the test utils from the techdocs addon package here. Can you clarify please?